docs/pages/guides/pages/built-in-pages.mdx

```diff
@@ -3,22 +3,66 @@ import { Screenshot } from "@/components/Screenshot"
 
 # Built-in Pages
 
-Auth.js comes by default with a set of pages that are presented to the user as they go through their authentication journey (sign up, sign in, sign out, error, etc...). This is helpful so that you don't need to write those from scratch when using the library first time.
+Auth.js comes by default with a set of pages that are presented to the user as they go through their authentication journey (sign up, sign in, sign out, error, etc...). This is helpful so that you don't need to write those from scratch when using the library first time. The UI created is based on the providers specified in your configuration file.
+
+```ts filename="lib/providers.ts"
+import Github from "next-auth/providers/github"
+import Credentials from "next-auth/providers/credentials"
+import type { Provider } from "next-auth/providers"
+
+export const providers: Provider[] = [
+  Github,
+  Credentials({
+    credentials: {
+      username: { label: "Username" },
+      password: { label: "Password", type: "password" }
+    },
+    async authorize(credentials) {
+      // mplement some logic to check if the user exists.
+      return credentials
+    }
+  })
+]
+```
```

Comment:
Not sure that this example is very helpful or relevant here.

Reply:
My idea with this code is to show the setup of the providers needed to create the same UI as shown in the screenshot at the bottom of the section. With this in mind, I have added the properties for the credentials because, if these are not set, it will throw an error as it expects an object or formData. I believe it’s better to include this content in the same section to avoid repeating the code for each implementation across different frameworks.

Reply:
Hmm okay I see where you're coming from. However, I think folks looking into the pages will have at least 1 provider already setup, it's kind of the minimum required thing to use auth.js, ya know what I mean. And I just don't see a lot of value in repeating it again here. I think getting it to look 100% like the screenshot is not necessary